### PR TITLE
Partially translated Design.md

### PR DESCRIPTION
**Pull request by MikeChongCan**

### CODE REVIEW

**Comment by MikeChongCan** [comment]
@yume190, could you create a list of action items in https://github.com/XCodeBazelize/Bazelize/projects?type=beta ?

I would love to start coding a bit before revisit these docs.

File: docs/Design.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Bazelize
+
+---
+
+## `Bazelize` Objectives
+
+ 1. Migrating to `bazel` with minimal impact on existing `XCode` projects.
+     * See [Ref](#Ref)
+ 2. Migrating `xxx.xcodeproj` and its dependencies to `bazel`, for example, `pod`, `spm`.
+     * See [Dependecy](Dependecy_ZH.md)
```
Comment (yume190) [inline docs/Design.md]: please remove `* See [Dependecy](Dependecy_ZH.md)`

File: docs/Design.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+First, let's talk about the code part. Our code will be applied to special rules, such as `xxx_library`.
+
+(Currently, only `xxx_library` can be applied to the same language)
+
+We are currently focusing on `swift_library` and `objc_library` implementations.
+
+Fortunately, `XCode Target` seems to support only one language.
+
+> Except for application, we can use `bridge-header` or generated header `RxSwift-Swift.h`
```
Comment (yume190) [inline docs/Design.md]: sorry, generated header change to ${target_name}-Swift.h

see [swift_library](https://github.com/bazelbuild/rules_swift/blob/master/doc/rules.md#swift_library-generates_header)

Comment (MikeChongCan) [reply docs/Design.md]: Changed to `${target_name}-Swift.h`

Thanks!

**Pull request merged**